daprdocs/content/en/operations/configuration/use-custom-certificates.md

```diff
@@ -0,0 +1,93 @@
+---
+type: docs
+title: "How-To: Use custom certificates in the Dapr sidecar"
```

Comment:
Why do you use "Custom" in the title? What is custom about this? Should this just be how to load certificates into the Dapr sidecar?

Reply:
Well, the original feature was to be able to install "extra"/"custom" certificates in the sidecar (similar to bring-your-own-certs), and hence the name. But yes, we can drop the "custom". Editing it.